exposing-to-internet/README.md

```diff
@@ -0,0 +1,88 @@
+
+When following the Apigee X [provisioning wizard](https://cloud.google.com/apigee/docs/api-platform/get-started/wizard-select-project), you will be prompted to [configure access routing](https://cloud.google.com/apigee/docs/api-platform/get-started/configure-routing) for your newly created instance. If you choose the internal option, the instance is only accessible internally via your GCP VPC network. If you subsequently decide you wish to expose it externally, this sample shows you how to do so.
+
+## Northbound Routing With Private Service Connect (PSC)
+
+This sample makes use of GCP's [Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) (aka PSC) capability to connect an external load balancer to the Apigee X instance.  Apigee supports PSC for both [northbound](https://cloud.google.com/apigee/docs/api-platform/system-administration/northbound-networking-psc) (ingress) and [southbound](https://cloud.google.com/apigee/docs/api-platform/architecture/southbound-networking-patterns-endpoints) (egress) connectivity.  This sample only deals with use of PSC for northbound connections from the internet.
+
+Each Apigee X instance contains a PSC [Service Attachment](https://cloud.google.com/vpc/docs/about-vpc-hosted-services#service-attachments). Information about the attachment can be found on the [Instances](https://cloud.google.com/apigee/docs/api-platform/system-administration/instances) page in the Apigee management UI, or via the [`organizations.instances.get`](https://cloud.google.com/apigee/docs/reference/apis/apigee/rest/v1/organizations.instances/get) API method.
+
+Customers can connect an external load balancer to this attachment using a [PSC network endpoint group](https://cloud.google.com/load-balancing/docs/negs#psc-neg), or PSC NEG for short.  PSC provides a fully managed option to establish connectivity, which does not involve the use of network bridge VMs.   The high level architecture is depicted in the diagram below:
+
+![Architecture](https://cloud.google.com/static/apigee/docs/api-platform/images/psc-arch.png)
+
+## Prerequisites
+1. [Provision Apigee X](https://cloud.google.com/apigee/docs/api-platform/get-started/provisioning-intro) (you may configure to use internal access routing only)
+2. Make sure the following tools are available in your terminal's `$PATH` (Cloud Shell has these preconfigured)
+    * [gcloud SDK](https://cloud.google.com/sdk/docs/install)
+    * curl
+    * jq
+    * npm
+
```

Comment:
Dont the user need to have permissions (roles) to create Apigee, loadbalancer and other components? Do we know the list of roles one should have to run this?